build/commands/lib/util.js

```diff
@@ -405,7 +405,6 @@ const util = {
         [androidIconBaseSource]: [androidIconBaseDest],
         [androidResSource]: [androidResDest],
         [androidResTemplateSource]: [androidResTemplateDest],
-        [androidContentPublicResSource]: [androidContentPublicResDest],
```

Comment:
I suppose we don't need their definitions as well
```
const androidContentPublicResSource = path.join(config.braveCoreDir, 'content', 'public', 'android', 'java', 'res')
const androidContentPublicResDest = path.join(config.srcDir, 'content', 'public', 'android', 'java', 'res')
```
Do we not want to override Chromium's `ic_search` icon anymore? It can be used in Chromium's xmls that we reuse.

Reply:
I left those two line in case they are needed in the future but yeah I can remove them.

@deeppandya Is the `ic_search` png override the same as the new svg, or are there something else needed to be done?

Reply:
We discussed this in Dm.